protocol_driver_grpc_async_callback.cc

```diff
@@ -316,4 +323,167 @@ void ProtocolDriverGrpcAsyncCallback::ShutdownServer() {
   }
 }
 
+using grpc::Status;
+
+class CallData {
+ public:
+  CallData(Traffic::AsyncService* service, grpc::ServerCompletionQueue* cq,
+           std::function<std::function<void()>(ServerRpcState* state)>* handler,
+           DistbenchThreadpool* thread_pool)
+      : service_(service), cq_(cq),
+        handler_(handler), responder_(&ctx_),
+        status_(CREATE) {
+    CHECK(thread_pool);
+    thread_pool_ = thread_pool;
+    ProcessRpcFsm();
+  }
+
+  void Unref() {
```

Comment:
We need to document that before we forget... Add a comment to explain why it's needed